frontend/src/pages/AddThek.js

```diff
@@ -0,0 +1,172 @@
+    };
+    setLoading(true);
+    fetch(API_URL("bags"), options)
+      .then((res) => res.json())
+      .then((data) => {
+        if (data.success) {
+          batch(() => {
+            dispatch(theks.actions.setItems(data.response));
+
+            dispatch(theks.actions.setError(null));
+          });
+        } else {
+          batch(() => {
+            dispatch(theks.actions.setItems([]));
+
+            dispatch(theks.actions.setError(data.response));
+          });
+        }
+      })
+      .finally(() => setLoading(false));
+  };
+
+  return (
+    <>
+      <Box>
+        <Menu />
+        {loading && <Loader />}
+
+        <Form onSubmit={onFormSubmit}>
+          <label htmlFor="colourInput"></label>
+          <Select
+            id="colourInput"
+            value={colour}
+            onChange={(e) => setColour(e.target.value)}
+            required
+          >
+            <option disabled value="">
+              Main colour:
+            </option>
+            <option value="Blue">Blue</option>
+            <option value="Green">Green</option>
+            <option value="Pink">Pink</option>
+            <option value="Purple">Purple</option>
+            <option value="Black">Black</option>
+            <option value="Multi">Multi-coloured</option>
+          </Select>
+
+          <label htmlFor="locationInput"></label>
+          <Select
+            id="locationInput"
+            value={location}
+            onChange={(e) => setLocation(e.target.value)}
+          >
+            <option disabled value="">
+              Nearest city:
+            </option>
+            <option value="Zurich">Zurich</option>
+            <option value="Basel">Basel</option>
+            <option value="Geneva">Geneva</option>
+            <option value="Bern">Bern</option>
+            <option value="Luzern">Luzern</option>
+            <option value="Lugano">Lugano</option>
+          </Select>
+
+          <label htmlFor="ageInput"></label>
+          <Select
+            id="ageInput"
+            value={age}
+            onChange={(e) => setAge(e.target.value)}
+          >
+            <option disabled value="">
+              Age-range:
+            </option>
+            <option value="1stGrade">1st Grade, 6-9 years</option>
+            <option value="4thGrade">4th Grade, 9-12 years</option>
+          </Select>
+          <ButtonContainer>
+            <Button type="submit">Add bag</Button>
+            <Button>
+              <Press
+                to={`/member/${memberId}`}
+                params={(accessToken, memberId)}
+              >
+                My Profile
+              </Press>
+            </Button>
+            <Logout />
+          </ButtonContainer>
+
+          {errors && (
+            <p className="warning-login">
+              The Thek has not been added to the database
+            </p>
+          )}
+        </Form>
+      </Box>
+      <Footer />
```

Comment:
You seem to repeat yourself by rendering `<Footer />` in many components - it would be far better to render it one time in `App.js`, outside of `<Routes />` component